### PR TITLE
Improve meshery contribution flow guide

### PR DESCRIPTION
**Pull request by daksh-sagar**
Signed-off-by: Daksh Sagar <daksh.sagar@yahoo.com>

**Description**
Edited CONTRIBUTING.md file to make the setup process more comprehensible.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by daksh-sagar** [comment]
@Nikhil-Ladha Please have a look 🙂

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -36,110 +39,144 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
...
 ### Prerequisites for building Meshery in your development environment:
+
 1. `Go` version 1.11+ installed if you want to build and/or make changes to the existing code.
 1. `GOPATH` environment variable should be configured appropriately
 1. `npm` and `node` should be installed on your machine, preferably the latest versions.
 1. Fork this repository (`git clone https://github.com/layer5io/meshery.git`), clone your forked version of Meshery to your local, preferably outside `GOPATH`. If you happen to checkout Meshery inside your `GOPATH` and you have a version of `Go` prior to version 1.13, please set an environment variable `GO111MODULE=on` to enable GO Modules.
 
 #### Build and run Meshery server
+
 To build & run the Meshery server code, run the following command:
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:100]: The information from line: 106 can be moved here in a way like:
```Before building & runnig the Meshery server code, you need to setup the Meshery UI:
  <deps, export cmnd>, then the 
 Command to build & run Meshery server is,
 ```sh
 make run-local
```
```

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
^^

File: CONTRIBUTING.md (hunk 1 above)
Comment (daksh-sagar) [reply CONTRIBUTING.md]: I've done suggested changes in a way it felt appropriate to me. If you want me to make further changes, please let me know, and I'll do so.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -36,110 +39,144 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
...
 ```sh
 make run-local
 ```
 
 Any time changes are made to the GO code, you will have to stop the server and run the above command again.
-Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`. One thing to note, you might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
-After running Meshery server, you will need to select your **Cloud Provider** by navigating to `localhost:9081`. Only then you will be able to use the Meshery UI on port `3000`.
+Once the Meshery server is up and running, you should be able to access Meshery on your `localhost` on port `9081` at `http://localhost:9081`.
+
+**NOTE**: You might NOT see the [Meshery UI](#contributing-ui) until the UI code is built as well.
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:121]: This note is no more needed.

Comment (daksh-sagar) [reply CONTRIBUTING.md:121]: Alright !

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -36,110 +39,144 @@ Signed-off-by: Jane Smith <jane.smith@example.com>
...
+
+To access the Meshery UI Development Server port on `3000`, you will need to select your **Cloud Provider** by navigating to `localhost:9081` after running the Meshery server.
```
Comment (Nikhil-Ladha) [inline CONTRIBUTING.md:123]: Link the text `Development server` to the development server section.

Comment (daksh-sagar) [reply CONTRIBUTING.md]: Done 🙂

**Review by Nikhil-Ladha — APPROVED** [review]
Now we will have less (hopefully 0) people asking this question `Page not found` 😆 
Thank you @daksh-sagar

**Comment by Nikhil-Ladha** [comment]
Yeaa.... first contribution...rather an very helpful one 👌

**Pull request merged**